src/video/movies.cpp

```diff
@@ -215,23 +305,64 @@ uint32_t ffmpeg_prepare_movie(char *name, bool with_audio)
 			codec_ctx->pix_fmt,
 			movie_width,
 			movie_height,
-			AV_PIX_FMT_YUVJ420P,
-			SWS_FAST_BILINEAR | SWS_ACCURATE_RND,
+			//AV_PIX_FMT_YUVJ420P,// don't use yuvj target pixformats. swscale goes crazy and does extra, un-asked-for TV->PC conversions 
+			targetpixelformat,
+            //SWS_FAST_BILINEAR | SWS_ACCURATE_RND, // might need to go back to this if performance issues are encountered. Unlikely, I hope.
+            SWS_LANCZOS | SWS_ACCURATE_RND | SWS_FULL_CHR_H_INT,
 			NULL,
 			NULL,
 			NULL
 		);
 
-		// change the range of input data by first reading the current color space and then setting it's range as yuvj.
-    int dummy[4];
-    int srcRange, dstRange;
-    int brightness, contrast, saturation;
-    sws_getColorspaceDetails(sws_ctx, (int**)&dummy, &srcRange, (int**)&dummy, &dstRange, &brightness, &contrast, &saturation);
-    const int* coefs = sws_getCoefficients(SWS_CS_DEFAULT);
-    dstRange = srcRange = movie_jpeg_range = true; // this marks that values are according to yuvj
-    sws_setColorspaceDetails(sws_ctx, coefs, srcRange, coefs, dstRange, brightness, contrast, saturation);
+		// if we need a colorspace conversion, set it up here
+        // this would also be the place to set up color range conversion, if it worked -- which it doens't
+        if (!okcolorspace /*|| !fullrange_input*/ || yuvjfixneeded || isdeepandtv){
```

Comment:
If the flag is not used, feel free to remove it. Otherwise if this PR is a WIP, I'll convert it as a Draft.

Reply:
I was keeping it as context for the accompanying comment explaining why we're not simply requesting a color range conversion from swscale. I'll delete it and just expand the comment.